server2/Arcadia.Assistant/Arcadia.Assistant.Sharepoint/Models/SharepointDepartmentsCalendarsSettings.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Arcadia.Assistant.Sharepoint.Models
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Fabric.Description;
+    using System.Text.Json;
+
+    public class SharepointDepartmentsCalendarsSettings : ISharepointDepartmentsCalendarsSettings
+    {
+        public SharepointDepartmentsCalendarsSettings(ConfigurationSection configurationSection)
+        {
+            try
+            {
+                var val = configurationSection.Parameters["Value"].Value;
+                this.DepartmentsCalendars = JsonSerializer.Deserialize<IEnumerable<SharepointDepartmentCalendarMapping>>(val);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```

Comment:
Maybe it is better to use appropriate `ILogger` instead?

Reply:
Remove debug code.